Marlin/src/HAL/STM32/tft/tft_spi.cpp

```diff
@@ -217,14 +222,31 @@ void TFT_SPI::Transmit(uint16_t Data) {
   while (!__HAL_SPI_GET_FLAG(&SPIx, SPI_FLAG_TXE)) {}
   while ( __HAL_SPI_GET_FLAG(&SPIx, SPI_FLAG_BSY)) {}
 
-  if (TFT_MISO_PIN != TFT_MOSI_PIN)
-    __HAL_SPI_CLEAR_OVRFLAG(&SPIx);   // Clear overrun flag in 2 Lines communication mode because received is not read
+  #if TFT_MISO_PIN != TFT_MOSI_PIN
```

Comment:
I remember that I changed it to runtime if because some version of stm32duino has variants with pins defined as runtime constants (static const int) and not compiler constants (define).

Reply:
@jmz52 I can't do it now, but worth check if we have any env using any shipped variant pointing to stm32duino 1.9.0 release... or we should just guarantee that STM32 env points to GitHub 1.9.0 tag instead of the stm32duino pio package...